Libraries/LibGUI/GItemView.cpp

```diff
@@ -121,9 +121,12 @@ void GItemView::mousedown_event(GMouseEvent& event)
             m_rubber_band_origin = event.position();
             m_rubber_band_current = event.position();
         } else {
+            int numselected = selection().size();
```

Comment:
Why does this need to be a separate variable?
Unless we need to hold on to the value returned by `size()` here, we're just confusing the code by giving an alternate name to `selection.size()`.
Instead, we can just check `else if (selection.size() > 1)` below.

Reply:
Good point. Changed.